5.0/List-files.md

```diff
@@ -0,0 +1,23 @@
+## Command description
+This operation will analyze and find all filesystems in a media dump and list all the files that are contained in [[Filesystems-recognized-by-Aaru.md|supported filesystems]].
+
+## Command usage
+```Aaru -d [true/false] -v [true/false] filesystem list -h [true/false] -e [encoding] -l [true/false]``` 
```

Comment:
Could you please make changes to the docs so that the command examples uses lowercase `aaru` instead of `Aaru`?